packages/icons/manifest.json

```diff
@@ -1868,6 +1868,11 @@
       "category": "Product and feature",
       "size": "24"
     },
+    {
+      "name": "motorbike",
+      "category": "Product and feature",
```

Comment:
The icon is in the "Miscellaneous" category in Figma. Let's match that here (and re-order the entry accordingly).